moped-api/README.md

```diff
@@ -63,15 +63,23 @@ $ pip install -r requirements/development.txt
 
 This particular requirements file includes tools such as pytest that make development and unit testing a lot easier, but it also makes the api bulky. Do not bother in installing the production or staging requirement files, those are only meant for cloud deployments.
 
-Next, set up your [AWS config and credentials](https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html) files. You can obtain your credentials from the AWS console.
+Next, set up your [AWS config and credentials](https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html) files. You can obtain your credentials from the AWS console. Your credentials will need read access to the AWS secret manager. These are the env vars of concern:
+
+```shell
+AWS_DEFAULT_REGION="us-east-1"
+AWS_ACCESS_KEY_ID=<your-access-key-id>
+AWS_SECRET_ACCESS_KEY=<your-access-key-secret>
+```
+
+The app will use your credentials to fetch its environment from secret manager. It defaults to "staging", and you may need to manually patch in a local Hasura endpoint to run end to end tests.
```

Comment:
i know this isn't super helpful, but it's what i did. the API seems designed to take a local env configuration, but i just didn't take the time to figure out to make it work.